vue-components/src/components/Checkbox.vue

```diff
@@ -9,10 +9,13 @@
 			:checked="checked"
 			:disabled="disabled"
 		>
+		<slot v-if="this.textDirection == 'rtl'" name="suffix" />
 		<label
 			class="wikit-checkbox__label"
+			:class="[ `wikit-checkbox__label`, `wikit-checkbox__label--${textDirection}` ]"
 			:for="id"
 		>{{ label }}</label>
+		<slot v-if="this.textDirection == 'ltr'" name="suffix" />
```

Comment:
In the case of the checkbox, I put the `slot` outside of the `label`, since we don't want the checkbox to be checked when the user clicks on the icon (we want the popover to show). the `label` is tied to the `Checkbox` (`:for="id"`)